CadVlan/User/views.py

```diff
@@ -58,7 +58,7 @@ def ajax_ldap_pop_name_mail(request, cn):
         auth = AuthSession(request.session)
         ldap_user = Ldap(auth).get_user(cn)
         lists['ldap_name'] = ldap_user['givenName'] + ' ' + \
-            ldap_user['initials'] + ' ' + ldap_user['sn']
+            ldap_user.get('initials', '') + ' ' + ldap_user['sn']
```

Comment:
Good contribution. Looking at the way this string concatenation is done I would ask you if you could improve it using the string format style. For example: 
```python
mystr = "%s %s" % (value0, value1)
```